crates/rolldown/tests/esbuild/default/comment_preservation/artifacts.snap

```diff
@@ -203,17 +203,13 @@ console.log(require.resolve(
 	"foo"
 	/* after */
 ));
-let [] = [];
-let [] = [];
 let [ ...s] = [...s];
 let [ ...s2] = [...s2];
 let {} = {};
 let {} = {};
 let { ...s3 } = { ...s3 };
 let { ...s4 } = { ...s4 };
 let [x] = [x];
-let [x2] = [x2];
-let [x3] = [x3];
```

Comment:
https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.ts%00let+%5B%0A++%2F*+foo+*%2F%0A%5D+%3D+%5B%0A++%2F*+bar+*%2F%0A%5D%3B%0Alet+%5B%0A++%2F%2F+foo%0A%5D+%3D+%5B%0A++%2F%2F+bar%0A%5D%3B%0A%0Alet+%5Bx2%5D+%3D+%5Bx2%5D%3B%0Alet+%5Bx3%5D+%3D+%5Bx3%5D%3B%0A&b=%00file.js%00&o=%7B%0A++treeShaking%3A+true%2C%0A++external%3A+%5B%22c%22%2C+%22a%22%2C+%22b%22%5D%2C%0A%22bundle%22%3A+true%2C%0Aformat%3A+%22esm%22%0A%7D, should has same output with esbuild in `bundle` mode